src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs

```diff
@@ -469,5 +341,234 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
+                            release.Categories = release.Categories.Union(cats).ToList();
+                        }
+                    }
+
+                    value = release.Categories.ToString();
+                    break;
+                case "size":
+                    release.Size = ParseUtil.GetBytes(value);
+                    value = release.Size.ToString();
+                    break;
+                case "leechers":
+                    var leechers = ParseUtil.CoerceLong(value);
+                    leechers = leechers < 5000000L ? leechers : 0; // to fix #6558
+                    if (release.Peers == null)
+                    {
+                        release.Peers = (int)leechers;
+                    }
+                    else
+                    {
+                        release.Peers += (int)leechers;
+                    }
+
+                    value = leechers.ToString();
+                    break;
+                case "seeders":
+                    release.Seeders = ParseUtil.CoerceInt(value);
+                    release.Seeders = release.Seeders < 5000000L ? release.Seeders : 0; // to fix #6558
+                    if (release.Peers == null)
+                    {
+                        release.Peers = release.Seeders;
+                    }
+                    else
+                    {
+                        release.Peers += release.Seeders;
+                    }
+
+                    value = release.Seeders.ToString();
+                    break;
+                case "date":
+                    release.PublishDate = DateTimeUtil.FromUnknown(value);
+                    value = release.PublishDate.ToString(DateTimeUtil.Rfc1123ZPattern);
+                    break;
+                case "files":
+                    release.Files = ParseUtil.CoerceInt(value);
+                    value = release.Files.ToString();
+                    break;
+                case "grabs":
+                    release.Grabs = ParseUtil.CoerceInt(value);
+                    value = release.Grabs.ToString();
+                    break;
+                case "downloadvolumefactor":
+                    release.DownloadVolumeFactor = ParseUtil.CoerceDouble(value);
+                    value = release.DownloadVolumeFactor.ToString();
+                    break;
+                case "uploadvolumefactor":
+                    release.UploadVolumeFactor = ParseUtil.CoerceDouble(value);
+                    value = release.UploadVolumeFactor.ToString();
+                    break;
+                case "minimumratio":
+                    release.MinimumRatio = ParseUtil.CoerceDouble(value);
+                    value = release.MinimumRatio.ToString();
+                    break;
+                case "minimumseedtime":
+                    release.MinimumSeedTime = ParseUtil.CoerceLong(value);
+                    value = release.MinimumSeedTime.ToString();
+                    break;
+                case "imdb":
```

Comment:
need to add imdbid support as well

https://github.com/Jackett/Jackett/commit/f5802306fa1fe4edb4c7f997100a6caebe71e989#diff-76185f0a79a2c5123ec2731d21a9318e3d857edda80e5b02dbbd9e812fbcc00bR1534

ref 
https://github.com/Prowlarr/Prowlarr/issues/550

Reply:
Need is a strong word

Reply:
All the v3 indexers have those changes  and any new ones added may have those as well.

not adding it means additional work on maintaining the indexers to ensure those all get reverted now and on every cherry-pick going forward and are backed out going forward of any new indexers